pkg/contracts/sui/crypto.go

```diff
@@ -79,7 +79,7 @@ func SerializeSignatureECDSA(signature [65]byte, pubKey *ecdsa.PublicKey) (strin
 	}
 
 	serialized := make([]byte, 1+sigLen+pubKeyLen)
-	serialized[0] = flagSecp256k1
+	serialized[0] = FlagSecp256k1
```

Comment:
let's not export this const if possible.
```suggestion
	serialized[0] = flagSecp256k1
```

Reply:
a0e1be917